### DISCUSSION TITLE
Asus rog strix g15 keyboard and  bottom cover lightning not working

### DISCUSSION THREAD

**Discussion by emircandrn** [seerge/g-helper#376] [General]
I downloaded g helper for the first time on my asus rog strix g15(2022) model. but for some reason the lights won't turn on. how can i solve it?

**Comment by seerge** [comment]
What is "bottom cover"? Lightbar? Try checkboxes under "Extra" related to lightbar. What exact model do you have?

**Reply by emircandrn** [reply]
asus rog strix g15 G513RM 6800h rtx 3060 model.
A strip-shaped adjustable light at the bottom of the device on strix models. both it and the keyboard light won't turn on. I tried all the checkboxes but it is not resolved. I am sorry for my English. translation errors may be.

**Reply by seerge** [reply]
@emircandrn does armoury crate able to control it ? If so can you try OpenRGB and see if it sees it and controls?

**Reply by emircandrn** [reply]
Yes. I was able to turn on the lights with openrgb. how do i do this via g helper, without openrgb?It doesn't change from g helper. now i tried again.

**Reply by seerge** [reply]
Oke, that's a good sign.

You can use https://www.nirsoft.net/utils/usb_devices_view.html  to find all USB devices with manufacturer ID 0x0b05 (it's asus) and post all found IDs here (may be yours is not in the supported list in the app)

Next to that you can check this build (try checkboxes / brightness / modes)
[GHelper.zip](https://github.com/seerge/g-helper/files/11470289/GHelper.zip)
and post log from ``%appdata%\ghelper`` here

**Comment by emircandrn** [comment]
13.05.2023 19:04:36: ------------
13.05.2023 19:04:36: App launched: ROG Strix G513RM_G513RM :0.61.0.0
13.05.2023 19:04:37: AutoSetting for Online
13.05.2023 19:04:37: PerformanceMode = 0 : OK
13.05.2023 19:04:37: ASUS BIOS rejected fan curve, resetting mode to 0
13.05.2023 19:04:37: PerformanceMode = 0 : OK
13.05.2023 19:04:37: Eco flag : 0
13.05.2023 19:04:37: Mux flag : 1
13.05.2023 19:04:37: Monitor Power On
13.05.2023 19:05:14: BatteryLimit = 85 : OK
13.05.2023 19:05:14: BatteryLimit = 80 : OK
13.05.2023 19:07:48: WMI event 124
13.05.2023 19:07:49: WMI event 124
13.05.2023 19:08:09: WMI event 197
13.05.2023 19:08:10: WMI event 197
13.05.2023 19:08:10: WMI event 197
13.05.2023 19:08:11: WMI event 196
13.05.2023 19:08:12: WMI event 196
13.05.2023 19:08:12: WMI event 196
13.05.2023 19:08:15: WMI event 197
13.05.2023 19:08:15: WMI event 196
13.05.2023 19:08:31: WMI event 197
13.05.2023 19:08:31: WMI event 197
13.05.2023 19:08:31: WMI event 197
13.05.2023 19:08:32: WMI event 197
13.05.2023 19:08:50: WMI event 196
13.05.2023 19:08:50: WMI event 196
13.05.2023 19:08:50: WMI event 196
13.05.2023 19:08:51: WMI event 196
13.05.2023 19:08:51: WMI event 196
13.05.2023 19:16:06: WMI event 124
13.05.2023 19:17:05: GPUMux = 0 : OK
13.05.2023 19:19:19: ------------
13.05.2023 19:19:19: App launched: ROG Strix G513RM_G513RM :0.61.0.0
13.05.2023 19:19:20: AutoSetting for Online
13.05.2023 19:19:20: BatteryLimit = 80 : OK
13.05.2023 19:19:20: PerformanceMode = 0 : OK
13.05.2023 19:19:20: ASUS BIOS rejected fan curve, resetting mode to 0
13.05.2023 19:19:20: PerformanceMode = 0 : OK
13.05.2023 19:19:20: Eco flag : 0
13.05.2023 19:19:20: Mux flag : 0
13.05.2023 19:19:20: ScreenOverdrive = 1 : OK
13.05.2023 19:19:20: Monitor Power On
13.05.2023 19:21:03: WMI event 179
13.05.2023 19:21:04: WMI event 179
13.05.2023 19:21:05: WMI event 179
13.05.2023 19:21:05: WMI event 179
13.05.2023 19:21:06: WMI event 179
13.05.2023 19:21:06: WMI event 179
13.05.2023 19:21:06: WMI event 179
13.05.2023 19:21:07: WMI event 179
13.05.2023 19:21:07: WMI event 179
13.05.2023 19:21:07: WMI event 179
13.05.2023 19:21:08: WMI event 179
13.05.2023 19:21:08: WMI event 179
13.05.2023 19:21:08: WMI event 179
13.05.2023 19:21:09: WMI event 179
13.05.2023 19:21:09: WMI event 179
13.05.2023 19:21:09: WMI event 179
13.05.2023 19:21:10: WMI event 179
13.05.2023 19:21:10: WMI event 179
13.05.2023 19:21:12: WMI event 196
13.05.2023 19:21:13: WMI event 196
13.05.2023 19:21:14: WMI event 197
13.05.2023 19:21:14: WMI event 197
13.05.2023 19:21:14: WMI event 197
13.05.2023 19:36:32: PerformanceMode = 1 : OK
13.05.2023 19:36:32: ASUS BIOS rejected fan curve, resetting mode to 1
13.05.2023 19:36:32: PerformanceMode = 1 : OK
13.05.2023 19:38:39: WMI event 196
13.05.2023 19:38:39: WMI event 196
13.05.2023 19:38:40: WMI event 197
13.05.2023 19:38:40: WMI event 197
13.05.2023 19:38:40: WMI event 197
13.05.2023 19:47:02: PerformanceMode = 0 : OK
13.05.2023 19:47:02: ASUS BIOS rejected fan curve, resetting mode to 0
13.05.2023 19:47:02: PerformanceMode = 0 : OK
13.05.2023 20:05:56: WMI event 196
13.05.2023 20:05:57: WMI event 196
13.05.2023 20:05:57: WMI event 196
13.05.2023 20:05:57: WMI event 196
13.05.2023 20:28:00: Monitor Dimmed
13.05.2023 20:28:15: Monitor Power Off
13.05.2023 20:28:15: BatteryLimit = 80 : OK
13.05.2023 20:30:59: Monitor Power On
13.05.2023 20:30:59: AutoSetting for Online
13.05.2023 20:30:59: BatteryLimit = 80 : OK
13.05.2023 20:30:59: PerformanceMode = 0 : OK
13.05.2023 20:30:59: ASUS BIOS rejected fan curve, resetting mode to 0
13.05.2023 20:30:59: PerformanceMode = 0 : OK
13.05.2023 20:30:59: Eco flag : 0
13.05.2023 20:30:59: Mux flag : 0
13.05.2023 20:30:59: ScreenOverdrive = 1 : OK
13.05.2023 20:32:01: WMI event 197
13.05.2023 20:32:01: WMI event 197
13.05.2023 20:32:01: WMI event 197
13.05.2023 20:32:02: WMI event 197
13.05.2023 20:32:02: WMI event 196
13.05.2023 20:32:03: WMI event 196
13.05.2023 20:32:03: WMI event 196
13.05.2023 20:32:03: WMI event 196
13.05.2023 20:32:05: WMI event 197
13.05.2023 20:32:05: WMI event 197
13.05.2023 20:32:05: WMI event 196
13.05.2023 20:32:06: WMI event 196
13.05.2023 20:32:06: WMI event 196
13.05.2023 20:32:07: WMI event 197
13.05.2023 20:32:08: WMI event 197
13.05.2023 20:32:08: WMI event 197
13.05.2023 20:32:08: WMI event 197
13.05.2023 20:32:09: WMI event 196
13.05.2023 20:32:09: WMI event 196
13.05.2023 20:32:09: WMI event 196
13.05.2023 20:32:10: WMI event 196
13.05.2023 20:32:11: WMI event 197
13.05.2023 20:32:11: WMI event 197
13.05.2023 20:32:11: WMI event 197
13.05.2023 20:36:07: WMI event 196
13.05.2023 20:36:07: WMI event 196
13.05.2023 20:36:07: WMI event 196
13.05.2023 20:36:08: WMI event 196
13.05.2023 20:36:08: WMI event 196
13.05.2023 20:36:08: WMI event 196
13.05.2023 20:36:09: WMI event 196
13.05.2023 20:36:09: WMI event 196
13.05.2023 20:54:49: WMI event 197
13.05.2023 20:54:49: WMI event 197
13.05.2023 20:54:50: WMI event 197
13.05.2023 20:54:50: WMI event 197
13.05.2023 20:54:50: WMI event 196
13.05.2023 20:54:51: WMI event 196
13.05.2023 20:54:51: WMI event 196
13.05.2023 20:54:51: WMI event 196
13.05.2023 21:07:37: PerformanceMode = 2 : OK
13.05.2023 21:07:37: ASUS BIOS rejected fan curve, resetting mode to 2
13.05.2023 21:07:37: PerformanceMode = 2 : OK
13.05.2023 21:10:03: GET GPU CLOCKS: 0, 0
13.05.2023 21:10:15: Boost 0
13.05.2023 21:10:19: PerformanceMode = 0 : OK
13.05.2023 21:10:19: ASUS BIOS rejected fan curve, resetting mode to 0
13.05.2023 21:10:19: PerformanceMode = 0 : OK
13.05.2023 21:10:19: GET GPU CLOCKS: 0, 0
13.05.2023 21:10:20: PerformanceMode = 2 : OK
13.05.2023 21:10:20: FanCPU = 3C-41-42-46-47-4B-4C-62-03-0C-0C-16-16-22-22-29 : OK
13.05.2023 21:10:20: FanGPU = 3C-41-42-46-47-4B-4C-62-08-11-11-1D-1D-26-26-2D : OK
13.05.2023 21:10:20: Boost 0
13.05.2023 21:10:20: GET GPU CLOCKS: 0, 0
13.05.2023 21:10:23: PerformanceMode = 0 : OK
13.05.2023 21:10:23: FanCPU = 3A-3D-40-44-48-4D-51-62-08-11-16-1A-22-29-30-45 : OK
13.05.2023 21:10:23: FanGPU = 3A-3D-40-44-48-4D-51-62-0C-16-1D-1F-26-2D-34-4A : OK
13.05.2023 21:10:23: GET GPU CLOCKS: 0, 0
13.05.2023 21:10:23: PerformanceMode = 2 : OK
13.05.2023 21:10:23: FanCPU = 3C-41-42-46-47-4B-4C-62-03-0C-0C-16-16-22-22-29 : OK
13.05.2023 21:10:23: FanGPU = 3C-41-42-46-47-4B-4C-62-08-11-11-1D-1D-26-26-2D : OK
13.05.2023 21:10:23: Boost 0
13.05.2023 21:10:23: GET GPU CLOCKS: 0, 0
13.05.2023 21:10:36: PerformanceMode = 0 : OK
13.05.2023 21:10:36: FanCPU = 3A-3D-40-44-48-4D-51-62-08-11-16-1A-22-29-30-45 : OK
13.05.2023 21:10:36: FanGPU = 3A-3D-40-44-48-4D-51-62-0C-16-1D-1F-26-2D-34-4A : OK
13.05.2023 21:10:45: GET GPU CLOCKS: 0, 0
13.05.2023 21:10:51: Boost 1
13.05.2023 21:10:53: PerformanceMode = 2 : OK
13.05.2023 21:10:53: FanCPU = 3C-41-42-46-47-4B-4C-62-03-0C-0C-16-16-22-22-29 : OK
13.05.2023 21:10:53: FanGPU = 3C-41-42-46-47-4B-4C-62-08-11-11-1D-1D-26-26-2D : OK
13.05.2023 21:10:53: Boost 0
13.05.2023 21:10:53: GET GPU CLOCKS: 0, 0
13.05.2023 21:11:06: PerformanceMode = 0 : OK
13.05.2023 21:11:06: FanCPU = 3A-3D-40-44-48-4D-51-62-08-11-16-1A-22-29-30-45 : OK
13.05.2023 21:11:06: FanGPU = 3A-3D-40-44-48-4D-51-62-0C-16-1D-1F-26-2D-34-4A : OK
13.05.2023 21:11:06: Boost 1
13.05.2023 21:11:06: GET GPU CLOCKS: 0, 0
13.05.2023 21:11:07: PerformanceMode = 2 : OK
13.05.2023 21:11:07: FanCPU = 3C-41-42-46-47-4B-4C-62-03-0C-0C-16-16-22-22-29 : OK
13.05.2023 21:11:07: FanGPU = 3C-41-42-46-47-4B-4C-62-08-11-11-1D-1D-26-26-2D : OK
13.05.2023 21:11:07: Boost 0
13.05.2023 21:11:07: GET GPU CLOCKS: 0, 0
13.05.2023 21:11:23: GET GPU CLOCKS: 0, 0
13.05.2023 21:30:37: ------------
13.05.2023 21:30:37: App launched: ROG Strix G513RM_G513RM :0.61.0.0
13.05.2023 21:30:37: AutoSetting for Online
13.05.2023 21:30:37: BatteryLimit = 80 : OK
13.05.2023 21:30:37: PerformanceMode = 2 : OK
13.05.2023 21:30:37: FanCPU = 3C-41-42-46-47-4B-4C-62-03-0C-0C-16-16-22-22-29 : OK
13.05.2023 21:30:37: FanGPU = 3C-41-42-46-47-4B-4C-62-08-11-11-1D-1D-26-26-2D : OK
13.05.2023 21:30:37: Boost 0
13.05.2023 21:30:37: Eco flag : 0
13.05.2023 21:30:37: Mux flag : 0
13.05.2023 21:30:37: ScreenOverdrive = 1 : OK
13.05.2023 21:30:37: Monitor Power On
13.05.2023 21:31:11: GET GPU CLOCKS: 0, 0


13.05.2023 19:04:36: ------------
13.05.2023 19:04:36: App launched: ROG Strix G513RM_G513RM :0.61.0.0
13.05.2023 19:04:37: AutoSetting for Online
13.05.2023 19:04:37: PerformanceMode = 0 : OK
13.05.2023 19:04:37: ASUS BIOS rejected fan curve, resetting mode to 0
13.05.2023 19:04:37: PerformanceMode = 0 : OK
13.05.2023 19:04:37: Eco flag : 0
13.05.2023 19:04:37: Mux flag : 1
13.05.2023 19:04:37: Monitor Power On
13.05.2023 19:05:14: BatteryLimit = 85 : OK
13.05.2023 19:05:14: BatteryLimit = 80 : OK
13.05.2023 19:07:48: WMI event 124
13.05.2023 19:07:49: WMI event 124
13.05.2023 19:08:09: WMI event 197
13.05.2023 19:08:10: WMI event 197
13.05.2023 19:08:10: WMI event 197
13.05.2023 19:08:11: WMI event 196
13.05.2023 19:08:12: WMI event 196
13.05.2023 19:08:12: WMI event 196
13.05.2023 19:08:15: WMI event 197
13.05.2023 19:08:15: WMI event 196
13.05.2023 19:08:31: WMI event 197
13.05.2023 19:08:31: WMI event 197
13.05.2023 19:08:31: WMI event 197
13.05.2023 19:08:32: WMI event 197
13.05.2023 19:08:50: WMI event 196
13.05.2023 19:08:50: WMI event 196
13.05.2023 19:08:50: WMI event 196
13.05.2023 19:08:51: WMI event 196
13.05.2023 19:08:51: WMI event 196
13.05.2023 19:16:06: WMI event 124
13.05.2023 19:17:05: GPUMux = 0 : OK
13.05.2023 19:19:19: ------------
13.05.2023 19:19:19: App launched: ROG Strix G513RM_G513RM :0.61.0.0
13.05.2023 19:19:20: AutoSetting for Online
13.05.2023 19:19:20: BatteryLimit = 80 : OK
13.05.2023 19:19:20: PerformanceMode = 0 : OK
13.05.2023 19:19:20: ASUS BIOS rejected fan curve, resetting mode to 0
13.05.2023 19:19:20: PerformanceMode = 0 : OK
13.05.2023 19:19:20: Eco flag : 0
13.05.2023 19:19:20: Mux flag : 0
13.05.2023 19:19:20: ScreenOverdrive = 1 : OK
13.05.2023 19:19:20: Monitor Power On
13.05.2023 19:21:03: WMI event 179
13.05.2023 19:21:04: WMI event 179
13.05.2023 19:21:05: WMI event 179
13.05.2023 19:21:05: WMI event 179
13.05.2023 19:21:06: WMI event 179
13.05.2023 19:21:06: WMI event 179
13.05.2023 19:21:06: WMI event 179
13.05.2023 19:21:07: WMI event 179
13.05.2023 19:21:07: WMI event 179
13.05.2023 19:21:07: WMI event 179
13.05.2023 19:21:08: WMI event 179
13.05.2023 19:21:08: WMI event 179
13.05.2023 19:21:08: WMI event 179
13.05.2023 19:21:09: WMI event 179
13.05.2023 19:21:09: WMI event 179
13.05.2023 19:21:09: WMI event 179
13.05.2023 19:21:10: WMI event 179
13.05.2023 19:21:10: WMI event 179
13.05.2023 19:21:12: WMI event 196
13.05.2023 19:21:13: WMI event 196
13.05.2023 19:21:14: WMI event 197
13.05.2023 19:21:14: WMI event 197
13.05.2023 19:21:14: WMI event 197
13.05.2023 19:36:32: PerformanceMode = 1 : OK
13.05.2023 19:36:32: ASUS BIOS rejected fan curve, resetting mode to 1
13.05.2023 19:36:32: PerformanceMode = 1 : OK
13.05.2023 19:38:39: WMI event 196
13.05.2023 19:38:39: WMI event 196
13.05.2023 19:38:40: WMI event 197
13.05.2023 19:38:40: WMI event 197
13.05.2023 19:38:40: WMI event 197
13.05.2023 19:47:02: PerformanceMode = 0 : OK
13.05.2023 19:47:02: ASUS BIOS rejected fan curve, resetting mode to 0
13.05.2023 19:47:02: PerformanceMode = 0 : OK
13.05.2023 20:05:56: WMI event 196
13.05.2023 20:05:57: WMI event 196
13.05.2023 20:05:57: WMI event 196
13.05.2023 20:05:57: WMI event 196
13.05.2023 20:28:00: Monitor Dimmed
13.05.2023 20:28:15: Monitor Power Off
13.05.2023 20:28:15: BatteryLimit = 80 : OK
13.05.2023 20:30:59: Monitor Power On
13.05.2023 20:30:59: AutoSetting for Online
13.05.2023 20:30:59: BatteryLimit = 80 : OK
13.05.2023 20:30:59: PerformanceMode = 0 : OK
13.05.2023 20:30:59: ASUS BIOS rejected fan curve, resetting mode to 0
13.05.2023 20:30:59: PerformanceMode = 0 : OK
13.05.2023 20:30:59: Eco flag : 0
13.05.2023 20:30:59: Mux flag : 0
13.05.2023 20:30:59: ScreenOverdrive = 1 : OK
13.05.2023 20:32:01: WMI event 197
13.05.2023 20:32:01: WMI event 197
13.05.2023 20:32:01: WMI event 197
13.05.2023 20:32:02: WMI event 197
13.05.2023 20:32:02: WMI event 196
13.05.2023 20:32:03: WMI event 196
13.05.2023 20:32:03: WMI event 196
13.05.2023 20:32:03: WMI event 196
13.05.2023 20:32:05: WMI event 197
13.05.2023 20:32:05: WMI event 197
13.05.2023 20:32:05: WMI event 196
13.05.2023 20:32:06: WMI event 196
13.05.2023 20:32:06: WMI event 196
13.05.2023 20:32:07: WMI event 197
13.05.2023 20:32:08: WMI event 197
13.05.2023 20:32:08: WMI event 197
13.05.2023 20:32:08: WMI event 197
13.05.2023 20:32:09: WMI event 196
13.05.2023 20:32:09: WMI event 196
13.05.2023 20:32:09: WMI event 196
13.05.2023 20:32:10: WMI event 196
13.05.2023 20:32:11: WMI event 197
13.05.2023 20:32:11: WMI event 197
13.05.2023 20:32:11: WMI event 197
13.05.2023 20:36:07: WMI event 196
13.05.2023 20:36:07: WMI event 196
13.05.2023 20:36:07: WMI event 196
13.05.2023 20:36:08: WMI event 196
13.05.2023 20:36:08: WMI event 196
13.05.2023 20:36:08: WMI event 196
13.05.2023 20:36:09: WMI event 196
13.05.2023 20:36:09: WMI event 196
13.05.2023 20:54:49: WMI event 197
13.05.2023 20:54:49: WMI event 197
13.05.2023 20:54:50: WMI event 197
13.05.2023 20:54:50: WMI event 197
13.05.2023 20:54:50: WMI event 196
13.05.2023 20:54:51: WMI event 196
13.05.2023 20:54:51: WMI event 196
13.05.2023 20:54:51: WMI event 196
13.05.2023 21:07:37: PerformanceMode = 2 : OK
13.05.2023 21:07:37: ASUS BIOS rejected fan curve, resetting mode to 2
13.05.2023 21:07:37: PerformanceMode = 2 : OK
13.05.2023 21:10:03: GET GPU CLOCKS: 0, 0
13.05.2023 21:10:15: Boost 0
13.05.2023 21:10:19: PerformanceMode = 0 : OK
13.05.2023 21:10:19: ASUS BIOS rejected fan curve, resetting mode to 0
13.05.2023 21:10:19: PerformanceMode = 0 : OK
13.05.2023 21:10:19: GET GPU CLOCKS: 0, 0
13.05.2023 21:10:20: PerformanceMode = 2 : OK
13.05.2023 21:10:20: FanCPU = 3C-41-42-46-47-4B-4C-62-03-0C-0C-16-16-22-22-29 : OK
13.05.2023 21:10:20: FanGPU = 3C-41-42-46-47-4B-4C-62-08-11-11-1D-1D-26-26-2D : OK
13.05.2023 21:10:20: Boost 0
13.05.2023 21:10:20: GET GPU CLOCKS: 0, 0
13.05.2023 21:10:23: PerformanceMode = 0 : OK
13.05.2023 21:10:23: FanCPU = 3A-3D-40-44-48-4D-51-62-08-11-16-1A-22-29-30-45 : OK
13.05.2023 21:10:23: FanGPU = 3A-3D-40-44-48-4D-51-62-0C-16-1D-1F-26-2D-34-4A : OK
13.05.2023 21:10:23: GET GPU CLOCKS: 0, 0
13.05.2023 21:10:23: PerformanceMode = 2 : OK
13.05.2023 21:10:23: FanCPU = 3C-41-42-46-47-4B-4C-62-03-0C-0C-16-16-22-22-29 : OK
13.05.2023 21:10:23: FanGPU = 3C-41-42-46-47-4B-4C-62-08-11-11-1D-1D-26-26-2D : OK
13.05.2023 21:10:23: Boost 0
13.05.2023 21:10:23: GET GPU CLOCKS: 0, 0
13.05.2023 21:10:36: PerformanceMode = 0 : OK
13.05.2023 21:10:36: FanCPU = 3A-3D-40-44-48-4D-51-62-08-11-16-1A-22-29-30-45 : OK
13.05.2023 21:10:36: FanGPU = 3A-3D-40-44-48-4D-51-62-0C-16-1D-1F-26-2D-34-4A : OK
13.05.2023 21:10:45: GET GPU CLOCKS: 0, 0
13.05.2023 21:10:51: Boost 1
13.05.2023 21:10:53: PerformanceMode = 2 : OK
13.05.2023 21:10:53: FanCPU = 3C-41-42-46-47-4B-4C-62-03-0C-0C-16-16-22-22-29 : OK
13.05.2023 21:10:53: FanGPU = 3C-41-42-46-47-4B-4C-62-08-11-11-1D-1D-26-26-2D : OK
13.05.2023 21:10:53: Boost 0
13.05.2023 21:10:53: GET GPU CLOCKS: 0, 0
13.05.2023 21:11:06: PerformanceMode = 0 : OK
13.05.2023 21:11:06: FanCPU = 3A-3D-40-44-48-4D-51-62-08-11-16-1A-22-29-30-45 : OK
13.05.2023 21:11:06: FanGPU = 3A-3D-40-44-48-4D-51-62-0C-16-1D-1F-26-2D-34-4A : OK
13.05.2023 21:11:06: Boost 1
13.05.2023 21:11:06: GET GPU CLOCKS: 0, 0
13.05.2023 21:11:07: PerformanceMode = 2 : OK
13.05.2023 21:11:07: FanCPU = 3C-41-42-46-47-4B-4C-62-03-0C-0C-16-16-22-22-29 : OK
13.05.2023 21:11:07: FanGPU = 3C-41-42-46-47-4B-4C-62-08-11-11-1D-1D-26-26-2D : OK
13.05.2023 21:11:07: Boost 0
13.05.2023 21:11:07: GET GPU CLOCKS: 0, 0
13.05.2023 21:11:23: GET GPU CLOCKS: 0, 0
13.05.2023 21:30:37: ------------
13.05.2023 21:30:37: App launched: ROG Strix G513RM_G513RM :0.61.0.0
13.05.2023 21:30:37: AutoSetting for Online
13.05.2023 21:30:37: BatteryLimit = 80 : OK
13.05.2023 21:30:37: PerformanceMode = 2 : OK
13.05.2023 21:30:37: FanCPU = 3C-41-42-46-47-4B-4C-62-03-0C-0C-16-16-22-22-29 : OK
13.05.2023 21:30:37: FanGPU = 3C-41-42-46-47-4B-4C-62-08-11-11-1D-1D-26-26-2D : OK
13.05.2023 21:30:37: Boost 0
13.05.2023 21:30:37: Eco flag : 0
13.05.2023 21:30:37: Mux flag : 0
13.05.2023 21:30:37: ScreenOverdrive = 1 : OK
13.05.2023 21:30:37: Monitor Power On
13.05.2023 21:31:11: GET GPU CLOCKS: 0, 0
  


I didn't quite understand what to do. I'm not very knowledgeable. I hope it's true. file dont upload

**Reply by seerge** [reply]
@emircandrn i don’t see a single line from keyboard here, did you use build from here and did you try to set different keyboard backlit mode / + checkboxes under extra?

**Reply by emircandrn** [reply]
![Ekran görüntüsü 2023-05-13 221024](https://github.com/seerge/g-helper/assets/133527305/70b048b5-236e-4cd3-9e54-06cc77be43a3)
are you talking about this?

**Reply by seerge** [reply]
@emircandrn yes, everythuing under backlight in Extra + keyboard on main window should at least try to change keyboard settings (and therefore write a log entry)

[GHelper.zip](https://github.com/seerge/g-helper/files/11470407/GHelper.zip)

Can you try this build and post log again (you can drag & drop log to text window it will upload it)

**Reply by emircandrn** [reply]
[log.txt](https://github.com/seerge/g-helper/files/11470414/log.txt)
yes it worked. Now I have full control. but i think i have another problem. There was no problem when using armory create. When valorantta gets my resolution of 4:3, it throws it to the desktop. Is it conflicting with something in the G helper? The game works but every time I press it it crashes to the desktop.

**Reply by seerge** [reply]
@emircandrn ok, good :) reason was that your windows is in turkish ... and it was looking only into "HID" (human input device) list , which is named differntly there

Can you check if this build still works for you? : 
[GHelper.zip](https://github.com/seerge/g-helper/files/11470464/GHelper.zip)

As for second queastion - If you mean game (Valorant?) then no, it has nothing to do with the app :) check drives / system settings :)
